### PR TITLE
Lronac2pds App and Test Update

### PR DESCRIPTION
**Pull request by acpaquette**
App conversion for lronac2pds and test conversion to gtests

## Description
Updates the lronac2pds app format and converts all existing make tests into gtests.

## Related Issue
N/A

## Motivation and Context
For the push to convert apps and gtests

## How Has This Been Tested?
These are tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/Fixtures.cpp [hunk 1]
```diff
@@ -321,11 +321,16 @@ namespace Isis {
       cubeL = new Cube();
       cubeR = new Cube();
 
-      cubeLPath = tempDir.path() + "/observationPairL.cub";
+      cubeLPath = "/Users/acpaquette/Desktop/observationPairL.cub";
```
Comment (kaitlyndlee) [inline isis/tests/Fixtures.cpp]: I think you forgot to change the path back.

Comment (acpaquette) [reply isis/tests/Fixtures.cpp]: Every darn time. Thanks @kaitlyndlee 👍

File: isis/src/lro/apps/lronac2pds/lronac2pds.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#ifndef lronac2pds_h
+#define lronac2pds_h
+
+#include "Pvl.h"
+#include "UserInterface.h"
+
+namespace Isis{
+  extern void lronac2pds(UserInterface &ui, Pvl *log=nullptr);
```
Comment (Kelvinrr) [inline isis/src/lro/apps/lronac2pds/lronac2pds.h]: The app doesn't seem to use a log? If so, probably best to remove the option from the api.

**Review by Kelvinrr — COMMENTED** [review]
Minor api concern.

**Review by Kelvinrr — APPROVED** [review]
test failures seem to be from upstream, so I think this is good

**Pull request merged**